src/fides/api/service/privacy_request/request_runner_service.py

```diff
@@ -324,6 +324,13 @@ def run_privacy_request(
                 "Terminating privacy request {}: request canceled.", privacy_request.id
             )
             return
+
+        if privacy_request.deleted_at is not None:
+            logger.info(
+                "Terminating privacy request {}: request deleted.", privacy_request.id
+            )
+            return
```

Comment:
same here, I didn't think it made sense to run an access request for a deleted privacy reqyest